pkgs/top-level/all-packages.nix

```diff
@@ -26695,20 +26692,6 @@ with pkgs;
 
   mkchromecast = libsForQt5.callPackage ../applications/networking/mkchromecast { };
 
-  # Backwards compatibility.
-  mod_dnssd = apacheHttpdPackages.mod_dnssd;
```

Comment:
This broke building GNOME with aliases disabled, fixed in https://github.com/NixOS/nixpkgs/pull/278744

Reply:
Ofborg didn't catch it when calculating outpaths because of https://github.com/NixOS/nixpkgs/pull/271123#discussion_r1442134594 as well, by the way (I've been investigating this)

Reply:
I'm working on a fix

Reply:
https://github.com/NixOS/nixpkgs/pull/278777